src/inabox/host-services.js

```diff
@@ -25,8 +26,27 @@ const ServiceNames = {
   EXIT: 'host-exit',
 };
 
+export const HostServiceError = {
+  MISS_MATCH: 1, // The HostService impl mis-matches the environment,
```

Comment:
This should probably be spelled `MISMATCH` since "mismatch" is a word

Reply:
done (after merging with master)